src/test/java/com/awesomeorg/airlineservice/integration_tests/PassengerControllerIntegrationTest.java

```diff
@@ -0,0 +1,95 @@
+package com.awesomeorg.airlineservice.integration_tests;
+
+import com.awesomeorg.airlineservice.AbstractIntegrationTest;
+import com.awesomeorg.airlineservice.entity.Passenger;
+import com.awesomeorg.airlineservice.protocol.CreatePassengerRequest;
+import com.awesomeorg.airlineservice.protocol.UpdatePassengerRequest;
+import com.awesomeorg.airlineservice.service.PassengerService;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+
+@AutoConfigureMockMvc
+public class PassengerControllerIntegrationTest extends AbstractIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private PassengerService passengerService;
+
+    @Test
+    @SneakyThrows
+    @DirtiesContext
+    public void testRegisterPassenger() {
+        CreatePassengerRequest request = new CreatePassengerRequest();
+        request.setFirstName("John");
+        request.setLastName("Doe");
+        request.setPhoneNumber("555-123-4567");
+        request.setEmailAddress("john.doe@example.com");
+
+        given(passengerService.createPassenger(any(CreatePassengerRequest.class)))
+                .willReturn(new Passenger(request));
+
+        mockMvc.perform(MockMvcRequestBuilders.post("/passengers/register")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content("{\"firstName\":\"John\",\"lastName\":\"Doe\",\"phoneNumber\":\"555-123-4567\",\"emailAddress\":\"john.doe@example.com\"}"))
```

Comment:
Also why this? you have mapper, use it to convert java object into the json string